source/_integrations/freebox.markdown

```diff
@@ -45,37 +45,12 @@ You can find out your Freebox host and port by opening this address <http://mafr
 The returned JSON should contain an `api_domain` (`host`) and a `https_port` (`port`).
 Please consult the [API documentation](https://dev.freebox.fr/sdk/os/) for more information.
 
-### Via the frontend
+### Via the UI
 
 Menu: **Settings** -> **Devices & Services**. Search for "Freebox", add your host and port, click submit.
 
 If you add the integration for the first time, follow the instructions in the [Initial setup](#initial-setup) section.
 
```

Comment:
```suggestion
```
Is this section still needed? We have the config flow and the initial setup.

Reply:
Imo this can be deleted, yes